File: src/main/java/net/dblsaiko/illuminate/client/GameRendererExt.java
package net.dblsaiko.illuminate.client;

import net.dblsaiko.illuminate.client.render.PostProcess;
import net.minecraft.client.render.GameRenderer;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface GameRendererExt {
    @NotNull
    PostProcess postProcess();

    @Nullable
    LightContainer activeRenderLight();

    void setActiveRenderLight(LightContainer lc);

    static GameRendererExt from(GameRenderer self) {
        return (GameRendererExt) self;
    }
}

File: src/main/kotlin/client/GameRendererExt.kt
package therealfarfetchd.illuminate.client

import net.dblsaiko.illuminate.client.GameRendererExt
import net.minecraft.client.render.GameRenderer

val GameRenderer.postProcess
    get() = GameRendererExt.from(this).postProcess()

var GameRenderer.activeRenderLight
    get() = GameRendererExt.from(this).activeRenderLight()
    set(value) {
        GameRendererExt.from(this).setActiveRenderLight(value)
    }